pkg/types/types.go

```diff
@@ -234,39 +262,69 @@ func (s byStepTime) Less(i, j int) bool {
 	return s[i].StepTime < s[j].StepTime
 }
 
-func mergeMetrics(metrics []Metric) Metric {
+func getPointInconsistentValueCount(selectedValue float64, values []float64) int {
```

Comment:
It's not very clear what InconsistentValue means w/o reading function implementation. A potentially more obvious naming would be something like `areValuesEqual()`.

Reply:
Legit.